### DISCUSSION TITLE
Thank You

### DISCUSSION THREAD

**Discussion by gnollio** [appy-one/acebase#233] [General]
I just wanted to say thank you for creating this project. There aren't many options out there to allow turn-key performant database functionality. The speed and minimal setup is a life-saver for some of my projects.

If you ever desire to commercialize anything that interacts with AceBase I'd love to help out. Perhaps an Electron-based GUI product, a Blog/WebApp/CRM/Intranet tool that can be installed with one command and a license key, etc. 

I know OSS can sometimes be a pain financially. I just sponsored and can see there are hardly any sponsorships, which is a shame given the usefulness of this project.

**Comment by appy-one** [comment]
Thank you so much @gnollio for sponsoring, you are a hero! I will be working on offering cloud hosted instances shortly but am open to all other commercial opportunities, awesome you'd want to help with that! Definitely let me know your ideas!
